installer/install.py

```diff
@@ -268,42 +266,70 @@ def debug_wifi():
 
 report_success()
 
-debug_wifi()
+os.system("/userdata/x1plus/debug_wifi.sh")
+
+def download_firmware(update_url, dest_path):
+    headers = {
+        'User-Agent': f"Python-urllib/3.{sys.version_info.minor}"
+    }
```

Comment:
`urllib` is the wrong user agent here (and how we got ourselves into this problem to start with...).  How about `f"X1Plus/{cfw_version}"`?  Then we identify ourselves correctly.